accname/name/comp_tooltip.html

```diff
@@ -11,11 +11,38 @@
 </head>
 <body>
 
-  <p>Tests the <a href="https://w3c.github.io/accname/#comp_tooltip">#comp_tooltip</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
+<p>Tests the <a href="https://w3c.github.io/accname/#comp_tooltip">#comp_tooltip</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<a href="#" title="label" data-expectedlabel="label" data-testname="link label from tooltip" class="ex"><img src="#" alt=""></a>
+<a href="#" title="label" data-expectedlabel="label" data-testname="link with img with tooltip label" class="ex"><img src="#" alt=""></a>
+<a href="#" title="label" data-expectedlabel="label" data-testname="link with text with tooltip label" class="ex">content</a>
+<div title="label" data-expectedlabel="label" data-testname="div with text with tooltip label" class="ex">content</div>
+<img src="foo.jpg" title="label" data-expectedlabel="label" data-testname="img with tooltip label without alt" class="ex">
+<img src="foo.jpg" title="label" data-expectedlabel="broken" alt="broken" data-testname="img with tooltip label with alt" class="ex">
+<img src="foo.jpg" data-expectedlabel="broken" alt="broken" data-testname="img with tooltip label without title" class="ex">
 
-<!-- Todo: test all remaining cases of https://w3c.github.io/accname/#comp_tooltip -->
+<textarea title="enter text" data-expectedlabel="enter text" placeholder="textarea" data-testname="textarea with tooltip label" class="ex"></textarea>
+<input type="text" title="label" data-expectedlabel="label" placeholder="text" data-testname="text input with tooltip label" class="ex">
+<input type="email" title="label" data-expectedlabel="label" placeholder="email" data-testname="email input with tooltip label" class="ex">
+<input type="password" title="label" data-expectedlabel="label" placeholder="password" data-testname="password input with tooltip label" class="ex">
```

Comment:
@scottaohara please review. I think placeholder vs title precedence is [still undefined in AccName](https://github.com/w3c/accname/issues/17) but defined in HTML-AAM, is that correct? If so, this is probably okay as-is.

Reply:
@howard-e these test names should include mention of the placeholder attr, I think.

Reply:
> @howard-e these test names should include mention of the placeholder attr, I think.

Thanks! Done in https://github.com/web-platform-tests/wpt/pull/42093/commits/5fd64fdf381cbc74aa88d36a6a283a2b7cd00471